coderd/healthcheck/provisioner.go

```diff
@@ -86,6 +93,12 @@ func (r *ProvisionerDaemonsReport) Run(ctx context.Context, opts *ProvisionerDae
 		r.Error = ptr.Ref("error fetching provisioner daemons: " + err.Error())
 		return
 	}
+
+	// Ensure stable order for display and for tests
+	sort.Slice(daemons, func(i, j int) bool {
+		return daemons[i].Name < daemons[j].Name
```

Comment:
doublecheck: we don't need to sort by other fields? There is no way that the report will contain old entries for provisioners with the same name?

Reply:
The database schema requires each provisioner daemon to have a unique name.